Server/Systems/SendToClient.hpp

```diff
@@ -0,0 +1,73 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SendToClient
+*/
+
+#pragma once
+
+#include <algorithm>
+#include <concepts>
+#include <map>
+#include <typeindex>
+#include "../Components/NetworkClient.hpp"
+#include "../Components/Networkable.hpp"
+#include "World/World.hpp"
+
+#include "GameComponents/DestinationComponent.hpp"
+#include "GameComponents/EquipmentComponent.hpp"
+#include "GameComponents/InvinsibleComponent.hpp"
+#include "GameComponents/InvisibleComponent.hpp"
+#include "GameComponents/LifeComponent.hpp"
+#include "GameComponents/PositionComponent.hpp"
+#include "GameComponents/VelocityComponent.hpp"
+
+///@brief a static map that is used to know which ID is used for a component type for the RFC protocol
```

Comment:
Prefer using "in order to" instead of "that is used to"

Reply:
I do not see the point

Reply:
In the line I've selected